pkgs/development/python-modules/databricks-cli/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, click
+, requests
+, tabulate
+, six
+, configparser
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "databricks-cli";
+  version = "0.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ebf123b5567c06b7583688077120ead075ca06938b9995d4acafa97863ed8ff";
+  };
+
+  checkInputs = [
+    pytest
+  ];
+
+  checkPhase = "pytest tests";
+  # tests folder is missing in PyPI
```

Comment:
please check out from github so we can get some unit tests

Reply:
The tests require internet access: https://github.com/databricks/databricks-cli/blob/aad9f95bffbc607b02558b0a847041e9e8fe93c9/tests/runs/test_cli.py#L58. The package is needed in mlflow due to this import: https://github.com/mlflow/mlflow/blob/279669ff4d198b2c662e9ee60f8575e421b2ae91/mlflow/utils/databricks_utils.py#L7.

Reply:
do you have an example of how you would like to use mlflow? probably help me gauge what would be reflected in the expressions

Reply:
definitely, here's a basic one: https://gist.github.com/tbenst/e793704190322915e9641c64351322a2

Essentially, I use mlflow to track model performance (similar to tensorboard if you're familiar) and store model parameters. I have a box on aws running `mlflow server`, and then use `mlflow` in local python scripts to log to the server.

Really appreciate all of your help!

Reply:
should also mention that databricks-cli is only needed for using their SaaS offering of `mlflow server`. I don't use this, so at least as far as I'm concerned could just remove the import, but thought I'd leave it in in case another nix user wants it + only have goodwill towards DataBricks for open-sourcing everything.